0000-charity-welcome-doc.md

```diff
@@ -0,0 +1,34 @@
+# Introducing a new series: Post-Mortem Book Reports
+
+## Dear fellow systems engineers,
```

Comment:
Hmmm, I felt like "systems engineers" was more generic than operators.  Trying to encompass an audience of software engineers, DBAs, SREs, execs, basically anyone interested in systems quality and incident response.  Anyone else have thoughts?